packages/jest-worker/src/workers/processChild.ts

```diff
@@ -97,7 +98,21 @@ function reportSuccess(result: unknown) {
     throw new Error('Child can only be used on a forked process');
   }
 
-  process.send([PARENT_MESSAGE_OK, result]);
+  try {
+    process.send([PARENT_MESSAGE_OK, result]);
+  } catch (error: unknown) {
+    if (
+      error instanceof Error &&
```

Comment:
Maybe use https://nodejs.org/api/util.html#utiltypesisnativeerrorvalue instead of `instanceof`? Not sure if cross real is an issue here, but doesn't hurt

Reply:
Yes. Thanks. I agree. 
Fix https://github.com/jestjs/jest/pull/15191/commits/a9f6e5be1301895b38da8ef6175720bd60fd586a